app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/LinkPreviewViewHolder.java

```diff
@@ -77,6 +77,7 @@
     private Realm realm;
     private String url;
     private ChatMessage model;
+    private MetaData data;
```

Comment:
Why the need to save this?

Reply:
in the previous implementation and android studio showed errors and asked to convert it into a array element

Reply:
Why do you need to store it globally?

Reply:
same as above, it is inside a function and on declaring it in the function, android studio forces it to make a final array element

Reply:
That doesn't answer it. Why are you storing it globally, why do you need it outside the scope